loadgen/cmd/otelbench/README.md

```diff
@@ -140,9 +140,15 @@ If you want to contribute to any go files, you need to create a changelog entry:
 
 ## Create new release
 
-TODO
-<!-----
-Have a workflow that gets triggered by a version update (maybe the version from the MAKEFILE, maybe another option), and runs `$(CHLOGGEN) update --version $(VERSION)`, updating the CHANGELOG.md and deleting all changelog fragments + it pushes a new image to elastic container
-registry.
----->
+There are two ways you can trigger a new release:
+- Manually, by triggering the GH actions workflow `bump-otelbench`.
+- Automatically, by updating the VERSION field in the Makefile.
```

Comment:
Q:
- does updating the VERSION field also trigger `bump-otelbench`?
- automatic as in the workflow is run automatically? The trigger is still manual, by doing a code change on the VERSION field. Is the benefit that we can perform a release in the same code change PR, and avoiding to trigger bump-otelbench manually?

Reply:
> does updating the VERSION field also trigger bump-otelbench?

Yes. If the version increasing, then CHANGELOG is updated, a PR is opened, and then the docker image is released (once the PR is merged).

> automatic as in the workflow is run automatically? 

Yes.

> The trigger is still manual, by doing a code change on the VERSION field. Is the benefit that we can perform a release in the same code change PR, and avoiding to trigger bump-otelbench manually?

Yes.